apps/storybook/src/AxialSelectionTool.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import type { AxialSelectionToolProps, Selection } from '@h5web/lib';
+import { SelectionRect } from '@h5web/lib';
+import {
+  AxialSelectionTool,
+  Pan,
+  ResetZoomButton,
+  VisCanvas,
+  Zoom,
+} from '@h5web/lib';
+import { useThrottledState } from '@react-hookz/web';
+import type { Meta, Story } from '@storybook/react';
+
+import FillHeight from './decorators/FillHeight';
+import { getTitleForSelection } from './utils';
+
+const Template: Story<AxialSelectionToolProps> = (args) => {
+  const { modifierKey } = args;
+
+  const [activeSelection, setActiveSelection] = useThrottledState<
+    Selection | undefined
+  >(undefined, 50);
+
+  return (
+    <VisCanvas
+      title={getTitleForSelection(activeSelection?.data)}
+      abscissaConfig={{ visDomain: [-10, 0], showGrid: true }}
+      ordinateConfig={{ visDomain: [50, 100], showGrid: true }}
+    >
+      <Pan modifierKey={modifierKey === 'Control' ? undefined : 'Control'} />
+      <Zoom />
+      <ResetZoomButton />
+
+      <AxialSelectionTool
+        {...args}
+        onSelectionChange={setActiveSelection}
+        onSelectionEnd={() => setActiveSelection(undefined)}
+      >
+        {({ data: [dataStart, dataEnd] }) => (
+          <SelectionRect startPoint={dataStart} endPoint={dataEnd} />
```

Comment:
Removing the `selectionType` logic, since it makes no sense for the axial selection.